teams/vsa/onboarding/VSA Technical Orientation.md

```diff
@@ -1,354 +1,489 @@
+
+## Onboarding FAQs
+
+> After I install everything, where do I start?
+
+Answer:
+
+If you are a front-end developer, clone this [website](https://github.com/department-of-veterans-affairs/vets-website) and follow these [set up directions](https://github.com/department-of-veterans-affairs/vets-website/blob/master/README.md). I would try to geth the command `yarn build --buildtype=vagovdev` to work. This will run the app locally and use a dev server as the API. As a bonus, you should try to get the [API](https://github.com/department-of-veterans-affairs/vets-api) set up as well
+
+If you are a back-end developer, clone this [API](https://github.com/department-of-veterans-affairs/vets-api) and get things set up without docker. You might also want to try to get the [website](https://github.com/department-of-veterans-affairs/vets-website) set up as well.
+
+> I want to add a new \_\_\_\_, where do I start?
+
+Checkout out the VA getting start [site](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/getting-started/). There are two sections on the left, `VA.gov workflow` and `Common tasks`. read through those two sections and try to implement some ideas in a branch.
+
+> Where do I go with my initial technical questions about the site?
+
+First, go to your onboarding buddies, then [slack channels](#slack) that seem appropriate. Ask questions, this the time to really see what is going on.
+
+> Do we have a style guide?
+
+We use prettier and eslint to enforce the Airbnb style guide. We have customized it a little, but usually you editor should be configured to automatically apply those stylings.
+
+> What does the development process look like from start to finish?
+
+Read this [guide](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/getting-started/workflow/overview)
+
+> In the website, where is a good place to start understanding the structure of the app?
+
+For a good start, look at he `src/applications/vaos` and the `src/applications/search` directories. But overall the app is pretty solidly done.
+
+> Where can find mock data?
+
+There are two repositories. There is there the [mock data for the API](https://github.com/department-of-veterans-affairs/vets-api-mockdata) and then there is the [fake log in information](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/3a1b22e261185ef42f3f7f849024b759bcfa9dad/Administrative/vagov-users/staging-test-accounts.md)
+
+> What is the mission of this project?
+
+Check out this repo and read through it: https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/content. This really defines the voice and mission of the project and how we as tech team are trying to approach it.
+
+> I found something in this guide that was wrong/missing/could use improvement?
```

Comment:
good way to end the FAQ and whole document; let's be sure that future edits keep this the last item